classes/Memcache.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * Memcache wrapper
+ *
+ * @package TheyWorkForYou
+ */
+
+namespace MySociety\TheyWorkForYou;
+
+class Memcache {
+
+    public function __construct() {
+        global $memcache;
```

Comment:
Now this is a class, this should presumably be a private class attribute, not a global? :)